src/components/TodoComponents/TodoList.js

```diff
@@ -6,17 +6,41 @@ import Todo from './Todo'
 
 
 const TodoList = props => {
-    const sortedList = props.todos.sort((a, b) => a.completed - b.completed)
+    const sortedList = props.todos.sort((a, b) => b.uid - a.uid)
     return (
-        <div className="list-container">       
-            {sortedList.map(task => {
-               return (
-                    <Todo
-                    task={task}
-                    toggleTask={props.toggleTask}
-                    />
-               )
-            })}
+        <div className="list">
+            <div className = "incomplete-list">
+                <h1>Todo</h1>
+                <div className="list-container">       
+                    {sortedList.map(task => {
+                        if (!task.completed){
+                            return (
+                                <Todo
+                                task={task}
+                                toggleTask={props.toggleTask}
+                                />
+                            )
+                        }                        
+                    })}
+                </div>
+            </div>
+            <div className="complete-list">
+                <div className="header-row">
+                    <h1>Completed</h1>
+                </div>
+                <div className="list-container">
+                    {sortedList.map(task => {
+                            if (task.completed){
+                                return (
+                                    <Todo
+                                    task={task}
+                                    toggleTask={props.toggleTask}
+                                    />
+                                )
+                            }                        
```

Comment:
https://github.com/josephpak/React-Todo/blob/fdfdc338f42784e9f70add4f024a16393f1a5bd1/src/components/TodoComponents/TodoList.js#L8-L48

this is such a clever approach